packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -0,0 +1,374 @@
+  )
+
+const SchedulePreviewBody = decorate([
+  addSubscriptions(({ schedule }) => ({
+    lastRunLog: cb =>
+      subscribeBackupNgLogs(logs => {
+        let lastRunLog
+        for (const runId in logs) {
+          const log = logs[runId]
+          if (
+            log.scheduleId === schedule.id &&
+            (lastRunLog === undefined || lastRunLog.start < log.start)
+          ) {
+            lastRunLog = log
+          }
+        }
+        cb(lastRunLog)
+      }),
+  })),
+  ({ job, schedule, lastRunLog }) => (
+    <Ul>
+      <Li>
+        {schedule.name
+          ? _.keyValue(_('scheduleName'), schedule.name)
+          : _.keyValue(_('scheduleCron'), schedule.cron)}{' '}
+        <Tooltip content={_('scheduleCopyId', { id: schedule.id.slice(4, 8) })}>
+          <CopyToClipboard text={schedule.id}>
+            <Button size='small'>
+              <Icon icon='clipboard' />
+            </Button>
+          </CopyToClipboard>
+        </Tooltip>
+      </Li>
+      <Li>
+        <StateButton
+          disabledLabel={_('stateDisabled')}
+          disabledHandler={enableSchedule}
+          disabledTooltip={_('logIndicationToEnable')}
+          enabledLabel={_('stateEnabled')}
+          enabledHandler={disableSchedule}
+          enabledTooltip={_('logIndicationToDisable')}
+          handlerParam={schedule.id}
+          state={schedule.enabled}
+          style={{ marginRight: '0.5em' }}
+        />
+        {job.runId !== undefined ? (
+          <Tooltip content={_('temporarilyDisabled')}>
+            <span>
+              <ActionButton
+                btnStyle='danger'
+                // 2020-01-29 Job cancellation will be disabled until we find a way to make it work.
+                // See https://github.com/vatesfr/xen-orchestra/issues/4657
+                disabled
+                handler={cancelJob}
+                handlerParam={job}
+                icon='cancel'
+                key='cancel'
+                size='small'
+                tooltip={_('formCancel')}
+              />
+            </span>
+          </Tooltip>
+        ) : (
+          <ActionButton
+            btnStyle='primary'
+            data-id={job.id}
+            data-name={job.name}
+            data-schedule={schedule.id}
+            data-type={job.type}
+            handler={_runBackupJob}
+            icon='run-schedule'
+            key='run'
+            size='small'
+          />
+        )}{' '}
+        {lastRunLog !== undefined && (
+          <LogStatus log={lastRunLog} tooltip={_('scheduleLastRun')} />
+        )}
+      </Li>
+    </Ul>
+  ),
+])
+
+@addSubscriptions({
+  jobs: subscribeBackupNgJobs,
+  metadataJobs: subscribeMetadataBackupJobs,
+  schedulesByJob: cb =>
+    subscribeSchedules(schedules => {
+      cb(groupBy(schedules, 'jobId'))
+    }),
+})
+class JobsTable extends React.Component {
+  static contextTypes = {
+    router: PropTypes.object,
+  }
+
+  static propTypes = {
+    main: PropTypes.bool,
```

Comment:
What do you think about `mainView` instead of `main`? Do you have another proposal?